ntbs-service/Pages/Notifications/OverviewPartials/_ContactTracingOverviewPartial.cshtml

```diff
@@ -3,128 +3,133 @@
                 </div>
             </div>
         </nhs-grid-column>
-        <nhs-grid-column grid-column-width="OneThird">
+        <nhs-grid-column grid-column-width="OneHalf">
             <div class="notification-details-label"> Number of contacts with active TB </div>
             <div>
                 <div class="contact-tracing-row">
-                    <span class="contact-tracing-label"> Adults  </span>
+                    <span class="contact-tracing-label"> Adults </span>
                     <span class="contact-tracing-value"> @(Model.Notification.ContactTracing.AdultsActiveTB)</span>
                 </div>
                 <div class="contact-tracing-row">
-                    <span class="contact-tracing-label"> Children  </span>
+                    <span class="contact-tracing-label"> Children </span>
                     <span class="contact-tracing-value"> @(Model.Notification.ContactTracing.ChildrenActiveTB)</span>
 
                 </div>
                 <div class="contact-tracing-row">
-                    <span class="contact-tracing-label"> Total  </span>
+                    <span class="contact-tracing-label"> Total </span>
                     <span class="contact-tracing-value"> @(Model.Notification.ContactTracing.TotalContactsActiveTB) </span>
                 </div>
             </div>
         </nhs-grid-column>
-    </nhs-grid-row>
-    <nhs-grid-row classes="notification-overview-details">
-        <nhs-grid-column grid-column-width="OneThird">
+        <nhs-grid-column grid-column-width="OneHalf">
             <div class="notification-details-label"> Number of contacts with latent TB </div>
             <div>
                 <div class="contact-tracing-row">
-                    <span class="contact-tracing-label"> Adults  </span>
+                    <span class="contact-tracing-label"> Adults </span>
                     <span class="contact-tracing-value"> @(Model.Notification.ContactTracing.AdultsLatentTB)</span>
                 </div>
                 <div class="contact-tracing-row">
-                    <span class="contact-tracing-label"> Children  </span>
+                    <span class="contact-tracing-label"> Children </span>
                     <span class="contact-tracing-value"> @(Model.Notification.ContactTracing.ChildrenLatentTB)</span>
-
                 </div>
                 <div class="contact-tracing-row">
-                    <span class="contact-tracing-label"> Total  </span>
+                    <span class="contact-tracing-label"> Total </span>
                     <span class="contact-tracing-value"> @(Model.Notification.ContactTracing.TotalContactsLatentTB) </span>
                 </div>
             </div>
         </nhs-grid-column>
-        <nhs-grid-column grid-column-width="OneThird">
+    </nhs-grid-row>
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="OneHalf"><p></p></nhs-grid-column>

```

Comment:
Is this just intentionally keeping a blank space? Is that the plan for good, or is it a placeholder for something?

Reply:
As discussed offline, this was meant to be a `nbsp;` and current format was agreed with Nancy - for logical business reasons. So reluctant to give it a rearrange at this point.